ClientApp/viking-vault/src/UserDashboard/AutomaticPaymentList.tsx

```diff
@@ -102,16 +103,30 @@ class AutomaticPaymentList extends React.Component<any, IAutomaticPaymentsState>
         this.setState({payments : paymentList});
     }
 
+    toggleAutomaticPaymentState = (id: number, state:boolean) =>{
+        let automaticPayments = this.state.payments.map((automaticPayment:IAutomaticPayment) => {
```

Comment:
You could use the array function `.forEach` in here.

blah.forEach(item => {
   if(blah) {
       item.something = somethingElse;
   }
})